DB/src/main/java/io/deephaven/process/ProcessInfoConfig.java

```diff
@@ -44,7 +44,7 @@
      * The default {@link SplayedPath} root for {@link ProcessInfo#getHostPathInfo()}.
      */
     @SuppressWarnings("WeakerAccess")
-    public static final String HOST_PATH_INFO_DIR_DEFAULT = "/etc/sysconfig/illumon.d/host";
+    public static final String HOST_PATH_INFO_DIR_DEFAULT = "/etc/sysconfig/deephaven.d/host";
```

Comment:
I think in DHE there is accompanying code for /etc/sysconfig/illumon.d all over the place in the devops module.
Which we are not concerned with on the OSS side, I get that, since we don't have an installer.  But given that,
what is the point of this path then?  Does it exist in the generated docker container? (did it exist as illumon.d? do we need to change something to make deephaven.d work?)

Reply:
To be honest, I'm suspicious that it's utterly useless now. I'm renaming it in an attempt to purge the old name from more places.